website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -797,6 +829,53 @@ curl \
   https://app.terraform.io/api/v2/policy-sets/polset-3yVQZvHzf5j3WRJ1/relationships/workspaces
 ```
 
+## Attach a Policy Set to exclusions
```

Comment:
```suggestion
## Exclude a workspace from a policy set 
```

Reply:
On further thought, I think we should use "exclude"/ "reinclude" here to explain how a workspace is "Excluded from a policy set's enforcement", which can be undone by "reincluding a workspace to a policy set's enforcement".  What do you think?

Reply:
Or does this both "attach" and "exclude"? Sorry to nitpick, negative phrasing is so confusing by default 😅

Reply:
Exclude sounds good!